Tools/WebKitTestRunner/InjectedBundle/ios/EventSenderProxyIOS.mm

```diff
@@ -99,6 +99,10 @@
 {
 }
 
+void EventSenderProxy::monitorWheelEvents()
+{
```

Comment:
Does iOS not have a wheel event capability with external mice? Not important for this change, but perhaps a future thing.